lib/routes/sms.js

```diff
@@ -83,11 +86,11 @@ module.exports = (log, config, customs, sms) => {
         }
 
         function sendMessage (senderId) {
-          return sms.send(phoneNumber, senderId, messageId, acceptLanguage)
+          return sms.send(phoneNumber, senderId, templateName, acceptLanguage)
         }
 
         function logSuccess () {
-          return request.emitMetricsEvent(`sms.${messageId}.sent`)
+          return request.emitMetricsEvent(`sms.${templateName}.sent`)
```

Comment:
@davismtl asked that we list each of the possible template values in [metrics-events.md](https://github.com/mozilla/fxa-auth-server/blob/master/docs/metrics-events.md), worthwhile to do as a part of this PR?